client/app/helpers/AvailabilityHelper.js

```diff
@@ -0,0 +1,78 @@
+import { currentAndNextEvents } from './EventHelper';
+import { durationFromNow } from './DateHelper';
+import sortBy from 'lodash/sortBy';
+import moment from 'moment';
+
+export const AVAILABILITY = {
+  ALL_DAY_AVAILABLE: 0,
+  CURRENTLY_AVAILABLE: 1,
+  CURRENTLY_BUSY: 2
+};
+
+function eventsInConferenceRoom(events, conferenceRoomId) {
+  return events.filter(event => event.conference_room.id === conferenceRoomId);
+}
+
+function allDayAvailableProps(conferenceRoom) {
+  return { conferenceRoomTitle: conferenceRoom.title, availability: AVAILABILITY.ALL_DAY_AVAILABLE };
+}
+
+function currentlyAvailableProps(conferenceRoom, nextEvent) {
+  const startTime = moment(nextEvent.start.date_time);
+  const duration = durationFromNow(startTime);
+  return { conferenceRoomTitle: conferenceRoom.title, duration, availability: AVAILABILITY.CURRENTLY_AVAILABLE };
```

Comment:
What if you want some other conference room attributes in the future?
Maybe it would be a good idea to just merge conference room attributes with the duration and availability prop?
Or store the room `id` instead of the `title` so you can easily look up the appropriate object.